src/test/java/stepDefinition/GenericStepDef.java

```diff
@@ -38,6 +38,6 @@ public void userPopulatesFieldsWithValues(Map<String, String> data) {
     public void userIsSuccessfullyLoggedIn(String elementName, String extension) {
         waitForPageLoaded();
         waitVisibility(getElementByName(elementName.concat(extension)));
-        Assert.assertTrue(String.format("%s element is displayed", elementName), getElementByName(elementName.concat(extension)).isDisplayed());
+        assertTrue(String.format("%s element is not displayed", elementName), getElementByName(elementName.concat(extension)).isDisplayed());
```

Comment:
as you added a custom logging configuration, your message should be positive not negative otherwise when you will read logs you will have discrepancies